### PR TITLE
Rnode 0.9 update: Either.proto etc.

### LINKED ISSUE #60: Either, Any types in CasperMessage.proto
@sebtomba writes:

> It shouldn't be hard in other languages to process this type the same way a Google `Any` is handled. Please look at the grpc code in your favorite client language. -- https://github.com/rchain/rchain/pull/2202#discussion_r260655968

See also:

https://github.com/rchain/rchain/blob/5ce7e1dfb7f6a4586b6a23ddc524f1d8135acde3/models/src/main/protobuf/CasperMessage.proto

### PR DESCRIPTION
**Pull request by dckc**
fixes #60

### CODE REVIEW

File: src/rnodeAPI.js [hunk 1]
```diff
@@ -142,45 +139,34 @@ function RNode(grpc /*: grpcT */, endPoint /*: { host: string, port: number } */
...
+      out = await either(DeployServiceResponse, send(f => client.createBlock({}, f)));
+    }
+    return out.message;
   }
 
   /**
    * Creates a block on your node
    * @memberof RNode
-   * @return A promise for response message
+   * @return A promise for { .message }
```
Comment (dckc) [inline src/rnodeAPI.js]: oops; meant to revert that doc change

**Comment by dckc** [comment]
@cboscolo it took me a while, but I think this is done now.

File: src/rnodeAPI.js [hunk 2]
```diff
@@ -142,45 +139,34 @@ function RNode(grpc /*: grpcT */, endPoint /*: { host: string, port: number } */
...
+    const x = await px;
+    return eitherSync(cls, x);
   }
 
-  function deployResponse(responseP) {
-    return responseP.then((response) => {
-      if (!response.success) { throw new Error(response.message); }
-      return response.message;
-    });
+  async function eitherSync/*::<T>*/(cls /*: Decoder<T>*/, x /*: IEither*/) /*: T*/{
```
Comment (JoshOrndorff) [inline src/rnodeAPI.js]: Does this one need to be async?

**Review by JoshOrndorff — APPROVED** [review]
I've looked over all the changes, and seen travis tests pass. This looks ready to me. Great work.

File: src/rnodeAPI.js [hunk 3]
```diff
@@ -273,23 +259,20 @@ function RNode(grpc /*: grpcT */, endPoint /*: { host: string, port: number } */
    * JOINed set of names in the tuplespace
    * @memberof RNode
    * @param pars The names onwhich to listen
-   * @return promise for ContinuationsWithBlockInfo
+   * @return promise for DataWithBlockInfo
```
Comment (dckc) [inline src/rnodeAPI.js:262]: that looks suspect

File: src/rnodeAPI.js (hunk 2 above)
Comment (dckc) [reply src/rnodeAPI.js]: oops!

**Pull request merged**